cmd/root.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "jh_extract",
+	Short: "Manipulate nvc files",
+	Long: `based off jh_extract.py`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("durr")
+	},
```

Comment:
You can just omit `Run` from the root function; I'm pretty sure it'll print a help message if no subcommand is given in that case

Reply:
:boom:

Reply:
This still needs to be done